### DISCUSSION TITLE
SONOFF SNZB-02 temperature_calibration ignored

### DISCUSSION THREAD

**Discussion by gafain** [Koenkk/zigbee2mqtt#6559] [Devices]
In your manpages for sensor  SNZB-02 your write that I can add this: temperature_calibration 

I edit the file share/zigbee2mqtt/devices.yaml:
after the lines 
'0x00124b00226a2f3b':
  friendly_name: Thermo03
I add ; 
  temperature_calibration: -0.5

I reboot the home assistant but the temperature reported do not change.
What I make wrong ?

**Comment by lux73** [comment]
works for me on zigbee2mqtt:
![Bildschirmfoto 2021-03-04 um 13 01 33](https://user-images.githubusercontent.com/3129990/109961290-11ef0280-7cea-11eb-924c-01960da9572f.png)

**Reply by sjorge** [reply]
Same, I did have to wait for one update from the device before it got applied. Which makes sense.
